Logic/igtlioConnector.cxx

```diff
@@ -887,13 +1054,13 @@ bool igtlioConnector::HasDevice(igtlioDevicePointer d )
 }
 
 //---------------------------------------------------------------------------
-int igtlioConnector::SendMessage(igtlioDeviceKeyType device_id, igtlioDevice::MESSAGE_PREFIX prefix)
+int igtlioConnector::SendMessage(igtlioDeviceKeyType device_id, igtlioDevice::MESSAGE_PREFIX prefix/*=Device::MESSAGE_PREFIX_NOT_DEFINED*/)
 {
   igtlioDevicePointer device = this->GetDevice(device_id);
   if (!device)
     {
       vtkErrorMacro("Sending OpenIGTLinkMessage: " << device_id.type << "/" << device_id.name<< ", device not found");
-      return 1;
+      return 0;
```

Comment:
Was this an existing bug that you've fixed?

Reply:
Yes, I think this was an existing bug.

The return type is not documented, but to be consistent with the return type of this->SendData(), the return type should be 1 if successful, 0 otherwise.